.github/workflows/release-mia.yaml

```diff
@@ -2,63 +2,118 @@ name: Create MIA binary release
 
 # MIA releases are created for tags like `mia-0.1.0`
 on:
+  workflow_dispatch:
   push:
     tags:
       - "mia-[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Why this custom format?

Reply:
I introduced this convention here:

- #5 

to separate releases of MIA and MIA installer.

We could e.g. use `X.Y.Z` tags for MIA and `mia-installer-X.Y.Z` for installer instead, but I decided to go with explicit approach.